server/plugin/plugin.go

```diff
@@ -95,6 +96,8 @@ type Plugin struct {
 	poster      poster.Poster
 	flowManager *FlowManager
 
+	b *i18n.Bundle
```

Comment:
This names seems a bit cryptic to me. Maybe we can call it `localization` or `i18n`? @hanzei Thoughts on this?

Reply:
Yeah, `b` isn't quite telling. 2./5 on `bundle` as `i18n` is already a library name.